CHANGES.rst

```diff
@@ -0,0 +1,67 @@
+================
+ Change History
+================
+
+0.7.0 (unreleased)
+==================
+
+- Add support for PyPy, Python 2.7, and Python 3.
+  This requires the addition of the ``zodbpickle`` dependency, even on
+  Python 2.6.
+- Fixed the ``--days`` argument to ``multi-zodb-gc`` with recent
+  versions of ``persistent``.
+
+0.6.1 2012-10-08
```

Comment:
A more usual (and IMHO more readable) format is `0.6.1 (2012-10-08)`, but I suppose you only moved the changelog, not reformatted it.